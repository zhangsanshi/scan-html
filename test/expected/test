<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <title>a sample for scanHtml</title>

    <link rel="stylesheet" href="css/default.css">
    <link rel="stylesheet" href="css/default1.css">
    <link rel="stylesheet" href="css/common.css">

</head>
<body>
<p>this is index</p>

<script src="js/default.js"></script>


<script src="js/view.js"></script>
<script src="view.js"></script>
</body>
</html>