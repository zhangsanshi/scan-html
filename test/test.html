<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <title>a sample for scanHtml</title>

    <div class="scanHtml" data-merge="css/default.css">
        <link rel="stylesheet" href="css/layout.css">
        <link rel="stylesheet" href="css/common.css">
    </div>
    <div class="scanHtml" data-merge="css/default1.css" >
        <link rel="stylesheet" href="css/layout2.css" >
        <link rel="stylesheet" href="css/common2.css" >
    </div>
    <link rel="stylesheet" href="css/common.css" >

</head>
<body>
<p>this is index</p>

<div class="scanHtml" data-merge="js/default.js" >
    <script src="js/app.js" ></script>
    <script src="js/app1.js" ></script>
</div>
<div class="scanDelete">
    <script src="js/app2.js"></script>
</div>

<script src="js/view.js" ></script>
<script src="view.js" ></script>
</body>
</html>