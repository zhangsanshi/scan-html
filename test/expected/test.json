{
	"js": {
		"default": {
			"src": [
				"test/js/app.js",
				"test/js/app1.js"
			],
			"dist": "tmp/test/js/default.js"
		}
	},
	"css": {
		"default": {
			"src": [
				"test/css/layout.css",
				"test/css/common.css"
			],
			"dist": "tmp/test/css/default.css"
		},
		"default1": {
			"src": [
				"test/css/layout2.css",
				"test/css/common2.css"
			],
			"dist": "tmp/test/css/default1.css"
		}
	},
	"copy": {
		"js": [
			"test/js/view.js",
			"test/view.js"
		],
		"css": [
			"test/css/common.css"
		]
	}
}